lib/team.rb

```diff
@@ -0,0 +1,38 @@
+require './lib/player'
+
+class Team
+
+  attr_reader :country,
+              :players
+
+  def initialize(country)
+    @country = country
+    @players = []
+    @eliminated = false
+  end
+
+  def add_player(player)
+    @players << player
+  end
+
+  def eliminated?
+    @eliminated
+  end
+
+  def eliminated
```

Comment:
This totally works, and I love the logic; think about refactoring so that this method just always sets `@eliminated` to `true` as that will be the end result no mater what.